### ISSUE TITLE
Documentation index

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5429]
|[<img alt="mesozoic's avatar" src="https://avatars.githubusercontent.com/u/458347?s=50" width="50" height="50">](https://github.com/mesozoic)| @mesozoic reported|
|-|-|
|Trac ID|trac#5429|
|Type|task|
|Created|2011-12-10 18:53:10Z|
|Branch|https://github.com/twisted/twisted/tree/doc-index-5429|

Says Glyph: 

 1. Go to http://twistedmatrix.com/documents/current
 2. Be embarrassed on our behalf.

Let me see if I can improve that a little bit.

This is _different_ than the task to integrate documentation across all of the Twisted modules, so that there is a way to navigate from [Twisted Core Documentation](http://twistedmatrix.com/documents/current/core/howto/index.html) back to the documentation index.

<details><summary>Searchable metadata</summary>

```
trac-id__5429 5429
type__task task
reporter__mesozoic mesozoic
priority__normal normal
milestone__ 
branch__branches_doc_index_5429 branches/doc-index-5429
branch_author__mesozoic mesozoic
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1323543190000000 1323543190000000
changetime__1331706255000000 1331706255000000
version__None None
owner__mesozoic mesozoic
cc__jesstess
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="mesozoic's avatar" src="https://avatars.githubusercontent.com/u/458347?s=50" width="50" height="50">](https://github.com/mesozoic)<a name="note_1"></a>|@mesozoic commented|
|-|-|

(In [33317]) Branching to 'doc-index-5429'

**Comment by twisted-trac** [comment]
|[<img alt="mesozoic's avatar" src="https://avatars.githubusercontent.com/u/458347?s=50" width="50" height="50">](https://github.com/mesozoic)|@mesozoic removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)|@jesstess set owner to @jesstess|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_5"></a>|@jesstess set owner to @mesozoic|
|-|-|

Thanks for working on this, mesozoic! This is vastly improved from the current state of the world. A few comments:

 * I'd remove "Documentation" from all of the top-level links, e.g. "Twisted Conch Documentation" ==> "Twisted Conch", since it has both documentation and examples and "Documentation" is already used in a nested link.
 * There are some inconsistencies in the naming:
   * "Twisted.Subproject" and "Twisted Subproject" are variously used
   * "Twisted Words" is sometimes called "Twisted IM"
 * I might just call "Twisted Mail Tutorial: Building an SMTP Client from Scratch" "Twisted Mail Documentation" for parallelism
 * Upgrading is obsolete and shouldn't be linked anywhere (it should get removed, but that can happen in another ticket)

**Comment by twisted-trac** [comment]
|[<img alt="mesozoic's avatar" src="https://avatars.githubusercontent.com/u/458347?s=50" width="50" height="50">](https://github.com/mesozoic)<a name="note_6"></a>|@mesozoic removed owner|
|-|-|

Good suggestions, all of them. Duly committed. Ready for review again.

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)|@jesstess set owner to @jesstess|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_8"></a>|@jesstess set owner to @mesozoic|
|-|-|

This looks great, please merge!

**Comment by twisted-trac** [comment]
|[<img alt="mesozoic's avatar" src="https://avatars.githubusercontent.com/u/458347?s=50" width="50" height="50">](https://github.com/mesozoic)<a name="note_9"></a>|@mesozoic set status to `closed`|
|-|-|

(In [33398]) Merge doc-index-5429

Author: mesozoic
Reviewer: jesstess
Fixes: [#5429](https://github.com/twisted/twisted/issues/5429)

Add an index page to the documentation tree and make references to "Twisted Products" within the documentation more consistent.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_10"></a>|@exarkun commented|
|-|-|

This was a duplicate of [#5419](https://github.com/twisted/twisted/issues/5419).  I marked that one as the duplicate, though, since this one is closed already.
